test/puppetlabs/trapperkeeper/services/status/status_proxy_service_test.clj

```diff
@@ -161,3 +163,47 @@
             (doall (map #(http-client/get % common-ssl-config) good-non-status-endpoint-requests))
             (is (= (count good-non-status-endpoint-requests)
                   (deref non-status-endpoint-counter)))))))))
+
+(deftest invalid-proxy-config-throws
+  (testing "Missing proxy-target-url throws schema error"
+    (is (thrown-with-msg?
+          clojure.lang.ExceptionInfo
+          #"Value does not match schema.*:proxy-target-url"
+          (let [bad-config (update-in
```

Comment:
Nitpick, but I'd prefer if the generation of the bad config here and below was done before the test assertion, since it's the result of using that bad config that's being tested rather than the generation of that config.